modules/nw-cudn-cr.adoc

```diff
@@ -34,7 +34,7 @@ metadata:
 EOF
 ----
 
-. Create a request for either a `Layer2` or `Layer3` topology type cluster-wide user-defined network:
+. Create a cluster-wide user-defined network for either a `Layer2` or `Layer3` topology type:
```

Comment:
I am not sure if I would do this change. You can create a CUDN resource and not get the network, but we can also just consider it a CUDN failure.

Reply:
Can you explain what you mean by you could create the cr but not get the network? @palonsoro

Reply:
I mean that you can create the CUDN resource, but there is an error and the cluster-wide user-defined network is not actually provisioned or usable, even when the resource is created.